src/panels/GettingStartedPanel/GettingStartedSteps.tsx

```diff
@@ -0,0 +1,242 @@
+import React from 'react';
+import {
+  Anchor,
+  Center,
+  Group,
+  Image,
+  Kbd,
+  List,
+  Stack,
+  Text,
+  Title
+} from '@mantine/core';
+
+import { FocusIcon, SceneIcon } from '@/icons/icons';
+import { IconSize } from '@/types/enums';
+
+import { OriginPanelMenuButton } from '../OriginPanel/OriginPanelMenuButton';
+
+import { AltitudeMouse } from './MouseDescriptions/AltitudeMouse';
+import { Mouse } from './MouseDescriptions/Mouse';
+import { MouseWithModifier } from './MouseDescriptions/MouseWithModifier';
+import { NavigationMouse } from './MouseDescriptions/NavigationMouse';
+import { AltitudeTask } from './Tasks/AltitudeTask';
+import { ChangeDeltaTimeTask } from './Tasks/ChangeDeltaTimeTask';
+import { SetBoolPropertyTask } from './Tasks/ChangePropertyTask';
+import { ChangeYearTask } from './Tasks/ChangeYearTask';
+import { ClearSkyButton } from './Tasks/Components/ClearSkyButton';
+import { CurrentAltitude } from './Tasks/Components/CurrentAltitude';
+import { CurrentFocus } from './Tasks/Components/CurrentFocus';
+import { CurrentLatLong } from './Tasks/Components/CurrentLatLong';
+import { FocusTask } from './Tasks/FocusTask';
+import { MarsTrailColorTask } from './Tasks/MarsTrailColorTask';
+import { NavigationTask } from './Tasks/NavigationTask';
+import { PauseTimeTask } from './Tasks/PauseTimeTask';
+
+export const GettingStartedSteps: React.JSX.Element[] = [
+  <Stack align={'center'}>
+    <Image src={'openspace-logo.png'} w={200} my={'lg'} />
```

Comment:
This logo does not load in OpenSpace

![image](https://github.com/user-attachments/assets/25699ac3-7069-4e48-8a20-af3b4cd4ce87)